bevrand.ocrapi/app.py

```diff
@@ -0,0 +1,98 @@
+from flask import Flask, jsonify, request
+from PIL import Image, ImageOps
+from error_model import InvalidUsage
+import pytesseract
+import base64
+import os
+import re
+
+application = Flask(__name__)
+
+
+@application.route('/ping')
+def ping_pong():
+    return 'pong'
+
+
+@application.route('/api/v1/base64', methods=['POST'])
+def parse_image_from_base64():
```

Comment:
Isn't this method missing argument(s)? Or is some python magic involved?

Reply:
POST do not have arguments. They receive a body